apps/system/test/unit/app_usage_metrics_test.js

```diff
@@ -49,11 +55,15 @@ suite('AppUsageMetrics:', function() {
       set: setNavigatorOnLine
     });
 
+    realPerformanceNow = window.performance.now;
+    window.performance.now = function() { return Date.now(); };
```

Comment:
I couldn't remember why this was necessary. Might be worth adding a comment that this is here because the test uses a fake clock.  And the fake performance.now could be made more realistic by setting taking Date.now() in the suiteSetup and subtracting it from the returned value. Doesn't actually matter, though.